src/test/java/urlshortener/web/UrlShortenerTests.java

```diff
@@ -132,6 +133,7 @@ public void thatShortenerFailsIfTheURLisNotSafe() throws Exception {
         .andExpect(status().isBadRequest());
   }
 
+  //ok
```

Comment:
?

Reply:
I was checking one by one if the tests were working I didn't realize I left that comment